### ISSUE TITLE
(Java) "Unknown packet with header (...) received" messages

### ISSUE THREAD

**Issue by wakachamo** [koraktor/steam-condenser#49]
Every time I perform an Rcon-related task using SourceServer (be it rconExec or rconAuth), I get the following:

```
  steamcondenser.PacketFormatException: Unknown packet with header -(random integer) received.
           at steamcondenser.steam.packets.rcon.RCONPacketFactory.getPacketFromData(Unknown Source)
           at steamcondenser.steam.sockets.RCONSocket.getReply(Unknown Source)
           at steamcondenser.steam.servers.SourceServer.rconAuth(Unknown Source)
```

It's a minor issue as it still performs the RCON command itself, however I never get a return value back, so the method is probably interrupting itself somewhere due to this exception. I can provide the remainder of the stack trace although it's most likely irrelevant as it pertains to my program itself. :P Thanks in advance.

**Comment by koraktor** [comment]
I'll check if I can reproduce this. Didn't touch the RCON code for quite some time now, so there might be changes in the protocol. Wouldn't be the first time.

**Comment by koraktor** [comment]
Sorry, almost forgot about this.
Just checked it myself. There seems to be some real problem here. I'll try to fix this ASAP.

**Comment by koraktor** [comment]
Okay this has been fixed in 7455fafbf9b5fe1cb0eca6dec5a7a5250466dd95 and version 0.11.1 respectively.

**Comment by wakachamo** [comment]
Many thanks :D
